src/renderer.js

```diff
@@ -260,6 +260,51 @@ class Renderer {
       }
     });
   }
+
+  printToPDF(url, options, config) {
+    return new Promise(async(resolve, reject) => {
+      function toNumber(string) {
+        const num = parseFloat(string);
+        return num !== NaN ? num : undefined;
+      }
+
+      function toBoolean(string) {
+        return string === '' || string === 'true' || string === true;
```

Comment:
why is empty string true

Reply:
It's fairly common to set a query parameter like `?landscape` intending to do `?landscape=true`. So this checks the query params object for the existence of the parameter.